api/src/main/java/org/openmrs/module/sync2/api/service/SyncPullService.java

```diff
@@ -6,11 +6,11 @@
 
 public interface SyncPullService {
 
-    AuditMessage pullDataFromParentAndSave(String category, Map<String, String> resourceLinks,
+    AuditMessage pullAndSaveFeedsFromParent(String category, Map<String, String> resourceLinks,
```

Comment:
This method will pull and save not atom feeds but the single OpenmrsObject (using atom feed).

Reply:
true